cmd/diff.go

```diff
@@ -108,27 +98,34 @@ func diffImages(image1Arg, image2Arg string, diffArgs []string) error {
 
 	diffTypes, err := differs.GetAnalyzers(diffArgs)
 	if err != nil {
-		glog.Error(err.Error())
-		return errors.New("Could not perform image diff")
+		return fmt.Errorf("Could not perform image diff: %s", err.Error())
```

Comment:
Here you usually want to just do something like `fmt.Errorf("error msg: %s", err)`
or `fmt.Errorf("error msg: %v", err)` if you might expect a typed error

Reply:
fixed